src/test/java/Homework17.java

```diff
@@ -0,0 +1,21 @@
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest{
+    @Test
+
+            public void addSongToPlaylist() throws InterruptedException {
+        String newSongAdded = "added 1 new song into";
+
+        navigateToPage();
+        provideEmail("heavenmayhem@gmail.com");
+        providePassword("Everything2Me0628!");
+        clickSubmit();
+        searchSong("Ketsa - That_s a Beat");
+        clickViewAll();
+        selectSong();
+        clickAddTo();
+        choosePlaylist();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

HINT

Please, add Assert

**One test - One Assert**